apps/dsaparam.c

```diff
@@ -25,12 +25,15 @@ NON_EMPTY_TRANSLATION_UNIT
 # include <openssl/x509.h>
 # include <openssl/pem.h>
 
+static int verbose = 0;
```

Comment:
Same as with genrsa, it would be nice if this option variable joined the other variables in `dsaparams_main`

Reply:
Hold on, neeeever mind that!  Since it's used in the callback as well, that makes this fine.  I take this change request back.